src/pages/resources/reference/push-notifications/templates/basic.md

```diff
@@ -0,0 +1,27 @@
+---
+title: AEP SDK basic push template
+description: Basic push template supported by Adobe Campaign Classic mobile extension.
+keywords:
+- Adobe Campaign Classic
+- Push
+- Push Template
+- Basic
+---
+
+import Tabs from './tabs/basic.md'
+
+# Push template - basic
+
+A standard notification including a title, subtitle, message body, optional image, icons, link, buttons.<br />Alternate title and body can optionally be specified for when the notification is expanded by the user.
```

Comment:
Same here - any reason for the br tag here?

Reply:
"An alternate title and body can be optionally specified when the notification..."